code/modules/logging/categories/log_category_misc.dm

```diff
@@ -0,0 +1,93 @@
+/datum/log_category/virus
```

Comment:
I don't love that there's a misc category, especially one this big. The debug/development logs (runtime/qdel/error etc) can probably split out and the rest alphabetized.